runtime/parser/transaction.go

```diff
@@ -52,15 +52,21 @@ func parseTransactionDeclaration(p *parser, docString string) *ast.TransactionDe
 
 	var parameterList *ast.ParameterList
 	if p.current.Is(lexer.TokenParenOpen) {
-		parameterList = parseParameterList(p)
+		parameterList, _ = parseParameterList(p)
```

Comment:
Does the error need to be handled here?

Reply:
Ah yes, this was automatically added (and hence ignored) by the IDE. Fixed it in 30f9a812d5725ccbbda8a4ca0a4c823738d709af